packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,232 @@
+      }
+      fontFamily="fontFamilyNotoSans"
+      href={href}
+      maxH={maxHeight[imagePosition]}
+      maxW={maxWidth[imagePosition]}
+      padding="space0"
+      textDecoration="none"
+    >
+      <Box.div
+        borderRadius="borderRadius40 borderRadius0 borderRadius0 borderRadius40"
+        lineHeight="lineHeight30"
+        padding="space70"
+      >
+        <Text.div
+          color="colorTextHeadingStronger"
+          fontSize="fontSize10"
+          lineHeight="lineHeight10"
+          marginBottom="space25"
+        >
+          {tag}
+        </Text.div>
+        <Heading marginBottom="space0" size="heading60">
+          {title}
+        </Heading>
+        <Text.p
+          color="colorTextStrongest"
+          fontSize="fontSize20"
+          marginBottom="space70"
+          marginTop="space30"
+        >
+          {text}
+        </Text.p>
+        {date && (
+          <Box.div
+            alignItems="center"
+            color="colorTextLinkStrong"
+            display="flex"
+            fontSize="fontSize20"
+            fontWeight="fontWeightMedium"
+            gap="space30"
+            marginBottom="space70"
+          >
+            <Icon decorative icon="calendar" size="sizeIcon30" />
+            <Text.span
+              fontFamily="fontFamilyNotoSans"
+              fontSize="fontSize20"
+              lineHeight="lineHeight30"
+            >
+              {date}
+            </Text.span>
+          </Box.div>
+        )}
+        <Box.div
+          alignItems={
+            isImageOnTop ? "center" : { _: "flex-start", md: "center" }
+          }
+          display="flex"
+          flexDirection={isImageOnTop ? "row" : { _: "column", md: "row" }}
+          flexShrink={2}
+          gap="space50"
+        >
+          {buttonText && (
+            <Box.div w={isImageOnTop ? "50%" : { _: "100%", md: "50%" }}>
+              <Button fullWidth onClick={onClickButton} variant="secondary">
+                {buttonText}
+              </Button>
+            </Box.div>
+          )}
+
+          {href ? null : <Anchor href={linkHref}>{linkText || ""}</Anchor>}
+        </Box.div>
+      </Box.div>
+      <Box.div
+        alignItems="center"
+        borderRadius={imageBorderRadius[imagePosition]}
+        display="flex"
+        maxH={isImageOnTop ? 180 : { _: 180, md: "unset" }}
+        maxWidth={isImageOnTop ? 400 : { _: "100%", md: 400 }}
+        overflow="hidden"
+        position="relative"
+      >
+        {iconUrl && (
+          <Box.div
+            alignItems="center"
+            backgroundColor="colorBackground"
+            borderRadius="borderRadiusCircle"
+            data-testid="service-content-card-icon"
+            display="flex"
+            gap="space30"
+            h="56px"
+            justifyContent="center"
+            left="16px"
+            position={"absolute"}
+            top={16}
+            w="56px"
```

Comment:
Why not using the tokens here?

Reply:
same as https://github.com/Localitos/pluto/pull/1209#discussion_r1443237021